packages/injective-math/src/fp_decimal/trigonometry.rs

```diff
@@ -0,0 +1,132 @@
+use num::pow::Pow;
+
+use crate::fp_decimal::FPDecimal;
+
+impl FPDecimal {
+    pub fn _cos(mut x: FPDecimal) -> FPDecimal {
+        x = FPDecimal::_change_range(x);
+        FPDecimal::_sin(FPDecimal::PI / FPDecimal::TWO - x)
+        /*
+        let sign = x.sign;
```

Comment:
why this commented out code?

Reply:
These are old implementation, no longer needed

Reply:
so remove it instead of commenting out ?